assets/js/settings/shared/index.js

```diff
@@ -9,6 +9,8 @@ import compareVersions from 'compare-versions';
 import { getSetting } from './get-setting';
 
 export * from './default-constants';
+export { default as defaultAddressFields } from './default-address-fields';
+export { default as countryAddressFields } from './country-address-fields';
```

Comment:
@nerrad I can certainly try. In case it wasn't clear, these had to be moved because they were consumed from multiple places which caused a CD issue. Settings is a shared location.

Reply:
Yup understood, but given the use of this package by other teams I think we should try to avoid this increase.

Reply:
The core locale was only used in one place so I moved it there https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4025/commits/2167f81d966d6ec0bc55f54d5fd4e8459f52d11f